app/models/mixins/virtual_total_mixin.rb

```diff
@@ -16,7 +16,7 @@ module ClassMethods
     #
     def virtual_total(name, relation)
       define_method(name) do
-        send(relation).count
+        send(relation).try(:size)
```

Comment:
Do you want me to put in a pr for number_of ?

```ruby
  def number_of(assoc)
    # ...
    send(assoc).try(:size)) || 0
  end
```

Heck, I'm even thinking it should be:

```ruby
  def number_of(assoc)
    # ...
    (respond_to?("total_#{assoc}") ? send("total_#{assoc}") : send(assoc).try(:size)) || 0
  end
```